app/javascript/src/utils/compiler.js

```diff
@@ -153,34 +153,166 @@ function extractAndInsertMixins(joinedItems) {
   return joinedItems
 }
 
-const regexRegex = /^\/(.+)\/(\w*)$/
-
-const conditionalOperations = {
-  // order is deliberate so longer operators match first
-  "is not": (l, r) => l !== r,
-  "is": (l, r) => l === r,
-  "contains": (l, r) => l.includes(r),
-  "test": (l, r) => {
-    const match = r.match(regexRegex)
-    if (!match) {
-      return false
+const comparisonOperators = {
+  "!": {
+    type: "unary-left",
+    order: -1,
+    eval: (v) => !v
+  },
+  "==": {
+    type: "binary",
+    order: 0,
+    eval: (l, r) => l === r
+  },
+  "!=": {
+    type: "binary",
+    order: 0,
+    eval: (l, r) => l !== r
+  },
+  "*=": {
+    type: "binary",
+    order: 0,
+    eval: (l, r) => l.includes(r)
+  },
+  "~=": {
+    type: "binary",
+    order: 0,
+    _regexRegex: /^\/(.+)\/(\w*)$/,
+    eval(l, r) {
+      const match = r.match(this._regexRegex)
+      if (!match) {
+        return false
+      }
+      const [_, pattern, flags] = match
+      let regex
+      try {
+        regex = new RegExp(pattern, flags)
+      } catch (err) {
+        return false
+      }
+      return regex.test(l)
     }
-    const [_, pattern, flags] = match
-    let regex
-    try {
-      regex = new RegExp(pattern, flags)
-    } catch (err) {
-      return false
+  },
+  "&&": {
+    type: "binary",
+    order: 1,
+    eval: (l, r) => l && r
+  },
+  "||": {
+    type: "binary",
+    order: 1,
+    eval: (l, r) => l || r
+  }
+}
+
+const sortedComparisonOperatorsSymbols = Object.keys(comparisonOperators)
```

Comment:
With the previous comment this could be a derived svelte store. But it being just a regular variable is fine too